lib/dishes.rb

```diff
@@ -0,0 +1,10 @@
+class Dish
+
+  attr_accessor :name, :price
+
+  def initialize(name, price)
+    @name = name
+    @price = price
+  end
+
```

Comment:
This works. If I am being very picky, I could suggest that you use an attr_reader and then manually create methods to allowing amending the name and price of a dish - this way you can control the input and ensure that e.g. the name is always a string, the price is always an integer.